freud/data.py

```diff
@@ -0,0 +1,138 @@
+               (Default value = 0.0).
+
+    Returns:
+        tuple (:class:`freud.box.Box`, :class:`np.ndarray`): freud Box,
+            particle positions, shape=(4*nx*ny*nz, 3)
+    """
+    fractions = np.array([[.5, .5, 0],
+                          [.5, 0, .5],
+                          [0, .5, .5],
+                          [0, 0, 0]], dtype=np.float32)
+    return make_cubic(nx, ny, nz, fractions, scale, noise)
+
+
+def make_bcc(nx=1, ny=1, nz=1, scale=1.0, noise=0.0):
+    """Make a body-centered-cubic crystal for testing.
+
+    Args:
+        nx: Number of repeats in the x direction, default is 1
+        ny: Number of repeats in the y direction, default is 1
+        nz: Number of repeats in the z direction, default is 1
+        scale: Amount to scale the unit cell by (in distance units)
+               (Default value = 1.0).
+        noise: Apply Gaussian noise with this width to particle positions
+               (Default value = 0.0).
+
+    Returns:
+        tuple (:class:`freud.box.Box`, :class:`np.ndarray`): freud Box,
+            particle positions, shape=(2*nx*ny*nz, 3)
+    """
+    fractions = np.array([[.5, .5, .5],
+                          [0, 0, 0]], dtype=np.float32)
+    return make_cubic(nx, ny, nz, fractions, scale, noise)
+
+
+def make_sc(nx=1, ny=1, nz=1, scale=1.0, noise=0.0):
+    """Make a simple cubic for testing.
+
+    Args:
+        nx: Number of repeats in the x direction, default is 1
+        ny: Number of repeats in the y direction, default is 1
+        nz: Number of repeats in the z direction, default is 1
+        scale: Amount to scale the unit cell by (in distance units)
+               (Default value = 1.0).
+        noise: Apply Gaussian noise with this width to particle positions
+               (Default value = 0.0).
+
+    Returns:
+        tuple (py:class:`freud.box.Box`, :class:`np.ndarray`): freud Box,
+            particle positions, shape=(nx*ny*nz, 3)
+    """
+    fractions = np.array([[0, 0, 0]], dtype=np.float32)
+    return make_cubic(nx, ny, nz, fractions, scale, noise)
+
+
+def make_square(nx=1, ny=1, fractions=np.array([[0, 0, 0]], dtype=np.float32),
+                scale=1.0, noise=0.0):
+    """Make a square crystal for testing
+
+    Args:
+        nx:
+            Number of repeats in the x direction, default is 1
+        ny:
+            Number of repeats in the y direction, default is 1
+        fractions:
+            The basis to replicate using the lattice.
+        scale:
+            Amount to scale the unit cell by (in distance units), default is
+            1.0
+        noise:
+            Apply Gaussian noise with this width to particle positions (Default
+            value = 0.0)
+
+    Returns:
+        tuple (py:class:`freud.box.Box`, :class:`np.ndarray`):
+            freud Box, particle positions, shape=(nx*ny*nz, 3)
+    """
+    box, positions = make_cubic(nx, ny, 1, fractions, scale, noise)
```

Comment:
Subtle thing: please verify that this is drawing from the appropriate distribution when considering noise. If it's drawing from a 3D gaussian and then setting z=0, I'm not sure if that's the same as drawing from a 2D gaussian to begin with.

Reply:
Overall we may need to verify if we're drawing x, y, z independently or not. We probably need to use the `multivariate_normal` function in both places: https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.random.multivariate_normal.html

Reply:
Assuming that the different dimensions are independent (i.e. assuming that the covariance matrix is diagonal), it is equivalent to sample them like this versus using the multivariate_normal function. The marginalization over each dimension trivially reduces in this case. However, there's no harm in using the multivariate_normal method in this case.